app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,50 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+
+    category = benchmark_run.result.keys[0]
```

Comment:
So let's say if there are two categories. Then do we need to check for both the categories and create an issue if any one of them fails? Basically checking in a loop for all the categories.